Client/Frontend/Strings.swift

```diff
@@ -9,7 +9,6 @@ import Foundation
  Strings with the following IDs may be removed after confirming with Product:
  - `ActivityStream.Pocket.SectionTitle`      --> replaced by `FirefoxHome.Pocket.SectionTitle`
  - `ActivityStream.Pocket.SectionTitle2`    --> replaced by `FirefoxHome.Pocket.SectionTitle`
```

Comment:
These two strings are actually unused. Who can sign off on removing them?

Reply:
@dnarcese 👀
Those strings refers to unused strings:
```
    public static let ASPocketTitle = MZLocalizedString(
        "ActivityStream.Pocket.SectionTitle",
        value: "Trending on Pocket",
        comment: "Section title label for Recommended by Pocket section",
        lastUpdated: .unknown)
    public static let ASPocketTitle2 = MZLocalizedString(
        "ActivityStream.Pocket.SectionTitle2",
        value: "Recommended by Pocket",
        comment: "Section title label for Recommended by Pocket section",
        lastUpdated: .unknown)
```

Reply:
Correct, what's unclear is why the note about product manager. 

Should I just remove them here?

Reply:
We wanna confirm with Product to make sure there's no plan to reuse those strings in the near future. Hence why confirming with @dnarcese is recommended

Reply:
Sounds good to me. Probably OK to merge this one then, and remove them afterwards? There are still a lot to remove… (need to update the list).

Reply:
They can be removed.